test/Elastic.Apm.AspNetCore.Static.Tests/Elastic.Apm.AspNetCore.Static.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
     <PropertyGroup>
-      <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+      <TargetFrameworks>netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
```

Comment:
Makes sense, thanks.
I'll try and remove it again.

Reply:
So, you were actually referring to the `Elastic.Apm.csproj`, correct @gregkalapos?
I think I added the additional `TargetFramework` because of this [line](https://github.com/elastic/apm-agent-dotnet/pull/1917/files#diff-f4eb846d047d2c3213ad755f354f6df40a9931c6baa56171373f7d44f78f6f66R74) in the project file. But we can probably work around that. Thoughts?

Reply:
🤦 Yes, I meant to add this comment to `Elastic.Apm.csproj` - sorry about the confusion, I simply added the comment to the wrong file. 

> I think I added the additional TargetFramework because of this [line](https://github.com/elastic/apm-agent-dotnet/pull/1917/files#diff-f4eb846d047d2c3213ad755f354f6df40a9931c6baa56171373f7d44f78f6f66R74) in the project file. But we can probably work around that. Thoughts?

Yes. Aren't those dependencies resolved at compile time? So what I'd expect happening is this: Let's say we want to support .NET 7 and our `Elastic.Apm` package ends up in a `net7.0` app. But we only compile for `net6.0` and older. When `Elastic.Apm` gets compiled, then one binary will be for `net6.0` and the line you referenced will match `'$(TargetFramework)' == 'net6.0'`, so it'll pull in the reference and generate the `.dll` from the project. And from that point the `net6.0` dll will have the `System.Diagnostics.PerformanceCounter` `Version="4.5.0"` dependency. If you pull in `Elastic.Apm.dll` into a `net7.0` app, that's still the same and it'll work. Or am I wrong? 🤔 

But then we have the whole startup hook thing on top of this which you mentioned and I have not thought about - so if this is a blocker for that, then it's maybe easier to directly target `net7.0`.

Reply:
Sure, scratch my last comment - that does not make sense. I had another (test) project file in mind when I wrote this.
I will remove the `net7.0` target here and add it back for the test project.
If we find out that we made are lives significantly harder with that when it comes to the startup hooks feature, we can revisit this decision. WDYT?

Reply:
> I will remove the net7.0 target here and add it back for the test project.
> If we find out that we made are lives significantly harder with that when it comes to the startup hooks feature, we can revisit this decision. WDYT?

Makes sense, agreed 👍